libsolidity/ast/AST.h

```diff
@@ -644,9 +644,9 @@ class InheritanceSpecifier: public ASTNode
 /**
  * Using for directive:
  *
- * 1. `using LibraryName for T` attaches all functions from the library `LibraryName` to the type `T`
- * 2. `using LibraryName for *` attaches to all types.
- * 3. `using {f1, f2, ..., fn} for T` attaches the functions `f1`, `f2`, ...,
+ * 1. `using LibraryName for T` binds all functions from the library `LibraryName` to the type `T`
+ * 2. `using LibraryName for *` binds to all types.
+ * 3. `using {f1, f2, ..., fn} for T` binds the functions `f1`, `f2`, ...,
  *     `fn`, respectively to `T`.
```

Comment:
I'd move this up so that it's all on a single line.

Reply:
Not sure I understand. Move which line up? And you mean squashing all 3 list items into a single line?

Reply:
Left a suggestion [here](https://github.com/ethereum/solidity/pull/13741#discussion_r1032686540) if it makes sense.

Reply:
Ah, I get it now. Fixed.